packages/flutter/lib/src/material/floating_action_button.dart

```diff
@@ -186,4 +206,81 @@ class _FloatingActionButtonState extends State<FloatingActionButton> {
 
     return result;
   }
+
+  @override
+  void didChangeDependencies() {
+    super.didChangeDependencies();
+    clearNotchMaker = Scaffold.setFloatingActionButtonNotchMakerFor(context, _notchMaker);
+  }
+
+  @override
+  void deactivate() {
+    if (clearNotchMaker != null)
+      clearNotchMaker();
+    super.deactivate();
+  }
+
+  Path _notchMaker(Rect host, Rect guest, Offset start, Offset end) {
+    assert(() {
+      if (end.dy != host.top)
+        throw new FlutterError(
+          'the floating action button\'s notch maker must only be used for a notch in the top edge.\n'
```

Comment:
should start with an uppercase letter (also applies to other error text below)

Reply:
done